src/crimson/osd/ec_backend.cc

```diff
@@ -38,4 +38,13 @@ ECBackend::submit_transaction(const std::set<pg_shard_t> &pg_shards,
   return make_ready_future<rep_op_ret_t>(seastar::now(), seastar::now());
 }
 
+PGBackend::get_attr_ierrorator::future<ceph::bufferlist>
+ECBackend::getxattr(
+  const hobject_t& soid,
+  std::string&& key) const
+{
+  // ENOSUPP! ECBackend reads xattr solely from the ObjectContext::attr_cache
```

Comment:
Should `get_attr_errorator` be expanded to include ENOSUPP?
`OpsExecuter::execute_op` handles all errors so it should return the correct error (enosupp).